continuous_integration/environment-3.10.yaml

```diff
@@ -62,6 +62,7 @@ dependencies:
   - psutil
   - requests
   - scikit-image
+  - imageio<2.28
```

Comment:
(Not a blocking comment)  But in the future it'd be good to link to a related issue, like https://github.com/dask/dask/issues/10215, so we know when we can remove the version restriction